data/language/nl-NL.txt

```diff
@@ -0,0 +1,2113 @@
+STR_0155    :Is hier al gebouwd
+STR_0156    :Kan {STRINGID2} hier niet kruisen
+STR_0157    :Kruisingen zijn niet mogelijk
+STR_0158    :Sein
+STR_0159    :Station
+STR_0160    :Vliegveld
+STR_0161    :Haven
+STR_0162    :Station in de weg
+STR_0163    :Sein in de weg
+STR_0164    :Kan vliegveld niet verwijderen...
+STR_0165    :Kan haven niet verwijderen...
+STR_0166    :Kan station niet verwijderen...
+STR_0167    :Verkeerde type spoor/weg!
+STR_0168    :Brugtypes moeten overeenkomen
+STR_0169    :Brug is niet geschikt voor een kruising
+STR_0170    :Spoorcombinatie niet mogelijk
+STR_0171    :Teveel objecten in het spel
+STR_0172    :Met de klok mee draaien
+STR_0173    :Tegen de klok in draaien
+STR_0174    :Locomotion: Eerste keer starten...
+STR_0175    :Locomotion: Bezig met checken van objecten...
+STR_0176    :Spel laden
+STR_0177    :Spel beëindigen
+STR_0178    :Spel beëindigen
+STR_0179    :Scenariobewerker verlaten
+STR_0180    :Landschap laden
+STR_0181    :Dit opslaan voordat je iets anders laadt?
+STR_0182    :Dit opslaan voordat je afsluit?
+STR_0183    :Dit opslaan voordat je afsluit?
+STR_0184    :Opslaan
+STR_0185    :Niet opslaan
+STR_0186    :Annuleren
+STR_0187    :OK
+STR_0188    :{SMALLFONT}{WHITE}{STRINGID2} naar links scrollen
+STR_0189    :{SMALLFONT}{WHITE}{STRINGID2} naar rechts scrollen
+STR_0190    :{SMALLFONT}{WHITE}{STRINGID2} snel naar links scrollen
+STR_0191    :{SMALLFONT}{WHITE}{STRINGID2} snel naar rechts scrollen
+STR_0192    :{SMALLFONT}{WHITE}{STRINGID2} naar links/rechts scrollen
+STR_0193    :{SMALLFONT}{WHITE}{STRINGID2} naar boven scrollen
+STR_0194    :{SMALLFONT}{WHITE}{STRINGID2} naar beneden scrollen
+STR_0195    :{SMALLFONT}{WHITE}{STRINGID2} snel naar boven scrollen
+STR_0196    :{SMALLFONT}{WHITE}{STRINGID2} snel naar beneden scrollent
+STR_0197    :{SMALLFONT}{WHITE}{STRINGID2} naar boven/beneden scrollen
+STR_0198    :Kaart
+STR_0199    :Voertuigenlijst
+STR_0200    :Lijst nieuwe voertuigen
+STR_0201    :Lijst
+STR_0202    :Bomenlijst
+STR_0203    :Orderlijst
+STR_0204    :Stationslijst
+STR_0205    :Stedenlijst
+STR_0206    :Industrielijst
+STR_0207    :Lijst nieuwe industrieën
+STR_0208    :Gebouwenlijst
+STR_0209    :Muurlijst
+STR_0210    :Waarderingslijst
+STR_0211    :Goederenlijst
+STR_0212    :Berichtenlijst
+STR_0213    :Bedrijvenlijst
+STR_0214    :Scenariolijst
+STR_0215    :Creditslijst
+STR_0216    :{WINDOW_COLOUR_2}{UP}{WHITE}  {CURRENCY}
+STR_0217    :{WINDOW_COLOUR_2}{DOWN}{WHITE}  {CURRENCY}
+STR_0218    :Kan niet overschakelen naar geselecteerde modus
+STR_0219    :{WINDOW_COLOUR_2}Kosten: {WHITE}{CURRENCY}
+STR_0220    :(speler 1)
+STR_0221    :(speler 2)
+STR_0222    :Kan kleurenschema niet veranderen...
+STR_0223    :Inzoomen
+STR_0224    :Uitzoomen
+STR_0225    :Steden
+STR_0226    :Industrieën
+STR_0227    :Vliegveld
+STR_0228    :Haven
+STR_0229    :Opties voor landschapsgenerator
+STR_0230    :
+STR_0231    :±
```

Comment:
Is this correctly encoded?

Reply:
It's the same in en-US. Most likely just a special code that IntelOrca has not converted, like {STRINGID}, {EURO}, etc.

Reply:
± has code 177 in CP1252, by the way.